sandpack-react/src/components/Preview/index.tsx

```diff
@@ -1,6 +1,7 @@
 import { useClasser } from "@code-hike/classer";
 import * as React from "react";
 
+import type { SandpackMessage } from "../../../../sandpack-client";
```

Comment:
```suggestion
import type { SandpackMessage } from "@codesandbox/sandpack-client";
```

Reply:
I've tried that at first, but VS Code shows the following error message:
> Could not find a declaration file for module '@codesandbox/sandpack-client'. '/home/peter/sandpack/sandpack-client/dist/cjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/codesandbox__sandpack-client` if it exists or add a new declaration (.d.ts) file containing `declare module '@codesandbox/sandpack-client';`ts(7016)

Reply:
I think you should run `yarn build` at first, in order to generate the types

Reply:
Thanks :)